numpy/add_newdocs.py

```diff
@@ -668,9 +668,8 @@ def luf(lamdaexpr, *args, **kwargs):
         will be in C-contiguous order (last-index varies the fastest).
         If order is 'F', then the returned array will be in
         Fortran-contiguous order (first-index varies the fastest).
-        If order is 'A' (default), then the returned array may be
-        in any order (either C-, Fortran-contiguous, or even discontiguous),
-        unless a copy is required, in which case it will be C-contiguous.
+        If order is 'A' (default), then the order of the returned array is
```

Comment:
No, "A" actually currently means: if no copy is made, nothing is done.

Copy True, output will be:
  - C: C-irder
  - F: F-order
  - A:  C order unless it is F order
  - K: Closest possible memory layout (fastest memory dimension should stay fastest).

If *no* copy is made:
  - C: C order is ensured, copy is made if necessary
  - F: F order is ensured, copy is made if necessary
  - A: *Nothing* is ensured (I would be very open to discuss changing this)
  - K: Nothing is ensured, like with A, no copy will be made even if the input is not a single blob of memory.

I meant "like A", because I thought it somewhat said that no memory order is ensured for no-copy mode.